_data/errors.yml

```diff
@@ -20,7 +20,7 @@ oidc:
         Check the `acr_values` parameter in your authentication request and ensure that you are passing either:
         - `http://idmanagement.gov/ns/assurance/ial/1` for Authentication Only applications, OR;
         - `http://idmanagement.gov/ns/assurance/ial/2` for Identity-Verification Permitted applications.
-        Learn more about IAL Service Level values here: [https://developers.login.gov/oidc/authorization/#service_level]https://developers.login.gov/oidc/authorization/#service_level){:target="_blank"}
+        Learn more about IAL Service Level values here: [https://developers.login.gov/oidc/authorization/#service_level](https://developers.login.gov/oidc/authorization/#service_level){:target="_blank"}
```

Comment:
Not directly relevant to the changes being made here, but why do we make this link open in a new window? (`target="_blank"`)

Related: https://www.w3.org/TR/WCAG20-TECHS/G200.html